packages/terra-clinical-result/src/proptypes/valuePropTypes.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+
+const valueQuantityPropShape = PropTypes.shape({
+  /**
+   *  Numeric Value for an Observation Result
+   */
+  value: PropTypes.number,
```

Comment:
```suggestion
  value: PropTypes. oneOfType([PropTypes.number, PropTypes.object, PropTypes.string]),
```

Then delete: valueNullPropShape &  valueStringPropShape